src/components/Expander/ExpanderGroup.tsx

```diff
@@ -82,43 +99,72 @@ const OpenAllButton = ({
   onClick,
```

Comment:
What is the difference between onClick and onClickAll? Do we need both for the API? If we do, I would at least consider the naming to be better descriptive of functionality.

Reply:
- `onClickAll` is not intended to be shown to "outside". It is a way to tell from core to components that _"Open/Close"_ all was clicked.
  - The `onClick` one is the one that components-level is used for handling the _"Open/Close all"_ . And it is not shown outside either.

Naming could be improved for sure, currently not having better and more describing ones in mind.

Reply:
Ah, so the onClickAll is a prop needed for the button to call a handler for the click event, while the onClick is a handler provided by the library user? If onClickAll is not visible on public API it's fine to leave it as is. It would also be possible to use just one onClick prop and wrap the user provided function inside another which could then do our magic before calling the user provided onClick.

Reply:
Is it currently still possible to provide a custom button for the open/close all? If it is, does this thing still work without the wrapper function as described above?

Reply:
Good find. 
It seems that even before this change, by testing on the `develop` branch, when providing custom `OpenAll` or `CloseAll` button the opening/closing all won't do anything.

Reply:
Let's leave it for another bug ticket. Feel free to resolve the discussion once the ticket is created.

Reply:
Bug ticket is now created to our system.